Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -59,11 +59,52 @@ public protocol HTTPNetworkTransportRetryDelegate: HTTPNetworkTransportDelegate
   ///   - request: The URLRequest which generated the error
   ///   - response: [Optional] Any response received when the error was generated
   ///   - retryHandler: A closure indicating whether the operation should be retried. Asyncrhonous to allow for re-authentication or other async operations to complete.
+  @available(*, deprecated, message: "Use networkTransport(_:receivedError:for:,response:continueHandler:) instead")
   func networkTransport(_ networkTransport: HTTPNetworkTransport,
                         receivedError error: Error,
                         for request: URLRequest,
                         response: URLResponse?,
                         retryHandler: @escaping (_ shouldRetry: Bool) -> Void)
+  
+  /// Called when an error has been received after a request has been sent to the server to see if an operation should be retried or not.
+  /// NOTE: Don't just call the `retryHandler` with `true` all the time, or you can potentially wind up in an infinite loop of errors
```

Comment:
This should be updated with the  appropriate `continueAction` that you shouldn't just call

Reply:
done